lib/api/units/validateBody.js

```diff
@@ -46,14 +45,14 @@ function parseContentType(contentType) {
  * @returns {boolean}
  */
 function isJsonContentType(contentType) {
-  // No need to explicitly check for "contentType" existence.
-  // In case given content type is invalid, "parseContentType"
-  // will return an error, which would be coerced to "false".
-  const [error, mediaType] = parseContentType(contentType);
-
-  // Silence an error on purporse because contentType
-  // may contain any kind of rubbish.
-  return error ? false : isJson(mediaType);
+  try {
```

Comment:
`parseContentType()` doesn't throw anymore, it returns `null` in case of problems, so this `try/catch` isn't necessary. `isJSON()` throws, because we feed it with `null` and it tries to do things like `.suffix` on `null`.

This could be cleaner:

```js
const mediaType = parseContentType(contentType);
return mediaType ? isJSON(mediaType) : false;
```

Reply:
Agree. Changed in 32b9f67.